New version: michidk.vscli version 1.3.0

michidk opened the pull request:
### Pull request has been created with [WinGet Updater](https://github.com/michidk/winget-updater) :rocket:

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/236233)

michidk committed:
New version: michidk.vscli version 1.3.0

stephengillie commented:
Hi @michidk,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 998.)
<!--
[Policy] Needs-Attention
[Policy] Validation-Defender-Error
-->

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2025-03-09 20:25:11.726 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerSharedLib\SHA256.cpp(144)\WindowsPackageManager.dll!00007FF94DD3273F: (caller: 00007FF94DD7AEC4) Exception(2) tid(1528) 8A150040  2025-03-09 20:25:12.879 [FAIL] Installer failed security check. Url: https://github.com/michidk/vscli/releases/download/v1.3.0/vscli-x86_64-pc-windows-msvc.zip Result: 0x80004005 MSI (s) (A4:E0) [20:25:08:494]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'. MSI (s) (A4:E0) [20:25:08:494]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'. MSI (s) (A4:E0) [20:25:08:494]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'. MSI (s) (A4:E0) [20:25:08:494]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'. MSI (s) (A4:E0) [20:25:08:494]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'. Property(S): OutOfDiskSpace = 0 Property(S): OutOfNoRbDiskSpace = 0 Property(S): PrimaryVolumeSpaceAvailable = 0 Property(S): PrimaryVolumeSpaceRequired = 0 Property(S): PrimaryVolumeSpaceRemaining = 0 MSI (s) (A4:04) [20:25:09:150]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'. MSI (s) (A4:04) [20:25:09:150]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'. MSI (s) (A4:04) [20:25:09:150]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'. MSI (s) (A4:04) [20:25:09:150]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'. MSI (s) (A4:04) [20:25:09:150]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

Automated error analysis suggests `00000000` (`0`)  (`4294967296`)  may translate to token `Success`, and may indicate "Success"

Automated error analysis suggests `00000000` (`0`)  (`4294967296`)  may translate to token `Success`, and may indicate "Success"

Automated error analysis suggests `00000000` (`0`)  (`4294967296`)  may translate to token `Success`, and may indicate "Success"

Automated error analysis suggests `00000000` (`0`)  (`4294967296`)  may translate to token `Success`, and may indicate "Success"

Automated error analysis suggests `00000000` (`0`)  (`4294967296`)  may translate to token `Success`, and may indicate "Success"

Automated error analysis suggests `80004005` (`-2147467259`)  (`2147500037`)  may translate to token `E_Fail`, and may indicate "Resource missing"

Automated error analysis suggests `8A15002D` (`-1978335187`)  (`2316632109`)  may indicate "Installer failed security check.  Another script should retry this about every 18 hours. <!-- [Policy] Validation-Defender-Error -->"

Automated error analysis suggests `8A150040` (`-1978335168`)  (`2316632128`)  may indicate "Error reading from the stream"

(Automated response - build 998.)

stephengillie commented:
@wingetbot run

stephengillie commented:
Hi @michidk,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 1000.)
<!--
[Policy] Needs-Attention
[Policy] Validation-Defender-Error
-->

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2025-03-10 08:03:37.388 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerSharedLib\SHA256.cpp(144)\WindowsPackageManager.dll!00007FFA513C273F: (caller: 00007FFA5140AEC4) Exception(2) tid(768) 8A150040  2025-03-10 08:03:38.775 [FAIL] Installer failed security check. Url: https://github.com/michidk/vscli/releases/download/v1.3.0/vscli-x86_64-pc-windows-msvc.zip Result: 0x80004005 MSI (s) (94:80) [08:03:33:230]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'. MSI (s) (94:80) [08:03:33:230]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'. MSI (s) (94:80) [08:03:33:230]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'. MSI (s) (94:80) [08:03:33:230]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'. MSI (s) (94:80) [08:03:33:230]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'. Property(S): OutOfDiskSpace = 0 Property(S): OutOfNoRbDiskSpace = 0 Property(S): PrimaryVolumeSpaceAvailable = 0 Property(S): PrimaryVolumeSpaceRequired = 0 Property(S): PrimaryVolumeSpaceRemaining = 0 MSI (s) (94:BC) [08:03:34:044]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'. MSI (s) (94:BC) [08:03:34:044]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'. MSI (s) (94:BC) [08:03:34:044]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'. MSI (s) (94:BC) [08:03:34:044]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'. MSI (s) (94:BC) [08:03:34:044]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

Automated error analysis suggests `00000000` (`0`)  (`4294967296`)  may translate to token `Success`, and may indicate "Success"

Automated error analysis suggests `80004005` (`-2147467259`)  (`2147500037`)  may translate to token `E_Fail`, and may indicate "Resource missing"

Automated error analysis suggests `8A15002D` (`-1978335187`)  (`2316632109`)  may indicate "Installer failed security check.  Another script should retry this about every 18 hours. <!-- [Policy] Validation-Defender-Error -->"

Automated error analysis suggests `8A150040` (`-1978335168`)  (`2316632128`)  may indicate "Error reading from the stream"

(Automated response - build 1000.)

stephengillie commented:
@wingetbot run

michidk commented:
hey @stephengillie, 
is there any documentation on what the root cause could be for this to fail?
This release (v1.3.0) updated a bunch of dependencies, added new arguments and mouse interactions to the terminal user interface.
I looked back at the changes and there is really nothing which should trigger a virus scanner.

Now the logs mention something about "OutOfDiskSpace", but I'm not sure what that means in this context.

To be fair, two other virus scanners detected a "trojan" as well: https://www.virustotal.com/gui/file/b17363b2eb69e681122da59cf0542d1d55225dd16966eca1b9bac11cfeb69a4b.
But this also applied to the previous version (v1.2.0): https://www.virustotal.com/gui/file/a4724e75c060ef843b4077988ef0be13acdac1fcca6cad86bfe5eb9b9b478012

stephengillie commented:
Hi @michidk,

We only consume the Defender output. I haven't worked with anyone on Defender teams, so I don't have insight as to why this update didn't pass on the first try.
